test/ProgressOnderwijsUtils.Tests/Data/DatabaseDescriptionTest.cs

```diff
@@ -110,6 +110,17 @@ public void CheckTableTriggers_works()
             PAssert.That(() => trigger.Definition == definition);
         }
 
+        [Fact]
+        public void SequencesAreLoadedAsExpected()
+        {
+            SQL($"create sequence dbo.TestSeq as Int minvalue 1").ExecuteNonQuery(Connection);
+            var db = DatabaseDescription.LoadFromSchemaTables(Connection);
+
+            var expected = new SequenceSqlDefinition("dbo.TestSeq", SqlSystemTypeId.Int, false, true, null, 1, 1, int.MaxValue);
+
+            PAssert.That(() => db.Sequences["DBO.TESTSEQ"] == expected);

```

Comment:
Die uppercase hier is opmerkelijk?

Reply:
Och, ik dacht direct even meenemen in de test dat de casing niet significant is.